src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -133,49 +156,20 @@ const TeamMembers = createReactClass({
     let access = this.getAccess();
 
     return (
-      <div>
-        <Panel>
-          <PanelHeader>
-            <PanelHeaderContentContainer>
-              <div>{t('Members')}</div>
-              {this.addMemberButton(access, params.orgId)}
-            </PanelHeaderContentContainer>
-          </PanelHeader>
-          <table className="table member-list">
-            <tbody>
-              {this.state.memberList.map((member, i) => {
-                return (
-                  <tr key={i}>
-                    <td className="table-user-info">
-                      <UserBadge user={member} orgId={params.orgId} />
-                    </td>
-                    <td>
-                      {access.has('org:write') ? (
-                        <Button
-                          size="small"
-                          onClick={this.removeMember.bind(this, member)}
-                        >
-                          {t('Remove')}
-                        </Button>
-                      ) : (
-                        <Tooltip
-                          title={t("You don't have have permission to remove members")}
-                        >
-                          <span>
-                            <Button size="small" disabled={true}>
-                              {t('Remove')}
-                            </Button>
-                          </span>
-                        </Tooltip>
-                      )}
-                    </td>
-                  </tr>
-                );
-              })}
-            </tbody>
-          </table>
-        </Panel>
-      </div>
+      <Panel>
+        <PanelHeader hasButtons disablePadding>
+          <StyledHeaderContainer>
+            <div>{t('Members')}</div>
+            {this.addMemberButton(access, params.orgId)}
+          </StyledHeaderContainer>
+        </PanelHeader>
+        {this.state.memberList.map((member, i) => (
```

Comment:
Do we need an empty state message?

Reply:
Can one have a team without any members? I'll check...

Reply:
the answer is yes. I will make one!

Reply:
actually, the answer is yes and no. it will be empty until you refresh the page, and then it will disappear :|

Reply:
ok empty message is added! it looks like we keep the team but don't send it over in the team list in settings which is maybe something to fix?